Engine/Alphas/ChartingInsightManagerExtension.cs

```diff
@@ -32,20 +32,24 @@ public class ChartingInsightManagerExtension : IInsightManagerExtension
         /// The string name used for the Alpha Assets chart
         /// </summary>
         public const string AlphaAssets = "Alpha Assets";
-
         private readonly bool _liveMode;
         private readonly StatisticsInsightManagerExtension _statisticsManager;
 
         private const int BacktestChartSamples = 1000;
         private DateTime _lastInsightCountSampleDateUtc;
         private DateTime _nextChartSampleAlgorithmTimeUtc;
+        private int _dailyInsightCount;
 
-        private readonly Chart _totalInsightCountPerSymbolChart = new Chart(AlphaAssets);          // Heatmap chart
-        private readonly Series _totalInsightCountSeries = new Series("Count", SeriesType.Bar, "#");
+        // Keep track, we only want to add the charts if the algorithm is producing insights
+        private bool _chartsAdded;
+        private IAlgorithm _algorithm;
 
-        private int _dailyCount;
+        // Heatmap chart
+        private readonly Chart _totalInsightCountPerSymbolChart = new Chart(AlphaAssets);          
         private readonly Dictionary<Symbol, int> _totalInsightCountPerSymbol = new Dictionary<Symbol, int>();
-        private readonly Dictionary<InsightScoreType, Series> _insightScoreSeriesByScoreType = new Dictionary<InsightScoreType, Series>();
+
+        private readonly Chart _totalInsightCountChart = new Chart("Insight Count");
```

Comment:
Unless required suggest avoiding changing the name of existing charts, can cause issues with any existing consumers

Reply:
All the names are the same! Just got a moved around a bit I promise 😁